packages/components/Molecules/Card/CardNavigationV2.vue

```diff
@@ -0,0 +1,346 @@
+<template>
+  <rpl-link v-if="link" :class="modifiers" :href="link.url" :innerWrap="false">
+    <div v-if="image" class="rpl-card-nav__image-wrapper">
+      <rpl-responsive-img class="rpl-card-nav__image" v-bind="image" alt="" :srcSet="srcSet" />
+    </div>
+    <div class="rpl-card-nav__content">
+      <div v-if="formattedDate || tag || status" class="rpl-card-nav__meta">
+        <div v-if="tag" class="rpl-card-nav__tag" >{{ tag }}</div>
+        <div v-if="status" class="rpl-card-nav__status" :class="`rpl-card-nav__status--${this.status.toLowerCase()}`">
+          <rpl-icon v-if="statusIcon" class="rpl-card-nav__status-icon" :symbol="statusIcon.symbol" :color="statusIcon.color" size="s" />
+          <span>{{ status }}</span>
+        </div>
+        <div v-if="formattedDate" class="rpl-card-nav__date">{{ formattedDate }}</div>
+      </div>
+      <h2 v-if="title" class="rpl-card-nav__title"><span>{{ trimmedTitle }}</span></h2>
+      <p v-if="summary" class="rpl-card-nav__summary">{{ trimmedSummary }}</p>
+      <p v-if="author" class="rpl-card-nav__author"><strong>Author:</strong> {{ author }}</p>
+    </div>
+  </rpl-link>
+</template>
+
+<script>
+import RplLink from '@dpc-sdp/ripple-link'
+import RplIcon from '@dpc-sdp/ripple-icon'
+import RplResponsiveImg from '@dpc-sdp/ripple-responsive-img'
+import formatdate from '@dpc-sdp/ripple-global/mixins/formatdate'
+import card from '@dpc-sdp/ripple-card/mixins/card'
+
+export default {
+  name: 'RplCardNavigationV2',
+  mixins: [formatdate, card],
+  components: {
+    RplLink,
+    RplResponsiveImg,
+    RplIcon
+  },
+  props: {
+    title: String,
+    image: Object,
+    summary: String,
+    link: Object,
+    tag: String,
+    dateStart: String,
+    dateEnd: String,
+    author: String,
+    status: String,
+    displayStyle: {
+      type: String,
+      default: 'noImage',
+      validator: val => ['noImage', 'thumbnail', 'featured'].includes(val)
+    }
+  },
+  data () {
+    return {
+      srcSet: [
+        { size: 'xs', height: 534, width: 764 },
+        { size: 's', height: 200, width: 764 },
+        { size: 'm', height: 232, width: 448 },
+        { size: 'l', height: 232, width: 333 }
+      ]
+    }
+  },
+  computed: {
+    modifiers () {
+      const prefix = 'rpl-card-nav'
+      const modifiers = [ prefix ]
+      modifiers.push(`${prefix}--${this.displayStyle.toLowerCase()}`)
+
+      return modifiers
+    },
+    formattedDate () {
+      if (!this.dateStart && !this.dateEnd) return ''
+
+      let formatted = this.formatDate(this.dateStart, 'DD MMMM YYYY')
+      if (this.dateEnd) {
+        formatted = this.formatDateRange(this.dateStart, this.dateEnd, { day: 'DD', month: 'MMM', year: 'YYYY' })
+      }
+
+      return formatted
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/scss/settings";
+@import "~@dpc-sdp/ripple-global/scss/tools";
+@import "scss/card";
```

Comment:
If there are any centralised style setting for all card types(e.g promo and nav), we can move to `scss/card.scss`.

Reply:
Good idea. I can see title, summary, date and tag/topic having the same styling. I have another PR to update Promo I'll do it in there.